### PR TITLE
RELATED: BB-2141 Document treatNullValuesAs property in filters

### PR DESCRIPTION
**Pull request by tomassatinsky**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/288/ : SUCCESS in 35s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/132/ : SUCCESS in 3s

File: docs/filter_visual_components.md [hunk 1]
```diff
@@ -263,6 +263,20 @@ This applies to the following types of measures:
 * Calculated measures with the percentage format set in the metadata catalog
 * Arithmetic measures with the `change` operator that has the percentage property `format` set
 
+#### Treat null as a number
```
Comment (evgeniabogranskaya) [inline docs/filter_visual_components.md:266]: Suggestion:

#### Treating null values as numbers
By default, any filter condition filters out null values. If you want a filter condition to treat null values as a number and to include them in calculations, add the optional `treatNullValuesAs` property to this condition and set it to that number.
For instance, you can treat null values as 0 to be accepted by a **less than 100** condition:

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/289/ : SUCCESS in 39s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/133/ : SUCCESS in 6s

**Comment by evgeniabogranskaya** [comment]
Language reviewed, LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/290/ : SUCCESS in 37s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/83/ : SUCCESS in 11s

**Pull request merged**